static/js/tools/map/util.ts

```diff
@@ -106,8 +106,8 @@ export const ALL_PLACE_CHILD_TYPES = {
 
 export const USA_CHILD_PLACE_TYPES = {
   Country: ["State", "County"],
-  State: ["County", "CensusZipCodeTabulationArea"],
-  County: ["County", "CensusTract", "CensusZipCodeTabulationArea"],
+  State: ["County", "City", "CensusZipCodeTabulationArea"],
```

Comment:
how's the loading time for drawing cities within a state?

Reply:
Between 3 and 5 seconds for California and Texas. We can consider caching / optimizing the geojsons if we want this to be faster?